neural_speed/models/model_utils/model_utils.h

```diff
@@ -49,7 +49,15 @@
 #define MODEL_SESSION_MAGIC MODEL_FILE_MAGIC_GGSN
 #define MODEL_SESSION_VERSION 1
 
-int64_t ns_log_level();
+inline int64_t ns_log_level() {
+  static int64_t log_level = -1;
+  if (log_level == -1) {
```

Comment:
meaningless?

Reply:
static variable log_level, when it is set to 0/1/2, we will ignore env